zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -279,7 +279,14 @@ function PageWrapper(props) {
                 onSubmit={handleSubmit}
                 ref={formRef}
               >
-                <FormControl className={clsx(common_classes.width100Percent, common_classes.displayFlex, common_classes.displayInlineFlex)} variant="outlined">
+                <FormControl
```

Comment:
All the changes in this file are linter changes am I right?

Reply:
yes they are linter changes that got made by the `prettier` extension I use so that the code runs error free like adding semicolon at the end just for good practice and writing each attribute on a next line to make it look neat. I can reverse them if there is an issue.

Reply:
I'll keep in mind to turn it off the next time I make any change.

Reply:
It's ok, we should keep it turned on 👍

Reply:
Cool I hope it can be merged then!